### PR TITLE
This commit will complete task 5 and all its relevant pass tests.

### PR DESCRIPTION
**Pull request by ruchikaroy**
Completed task 5 and added description to the api endpoint

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -80,3 +80,34 @@ describe("/api/articles/:article=id", () => {
...
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("GET api/articles should return 404 for non-existent endpoint", () => {
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:105]: good test! Maybe should sit under its own describe block.  Not necessarily to do with endpoint api/articles!

File: models/articles.model.js [hunk 2]
```diff
@@ -23,3 +23,21 @@ exports.fetchArticlesById = (id) => {
...
+  articles.title,topic,
+  articles.author,
+  articles.created_at,
+  articles.votes,
+  articles.article_img_url, 
+  CAST (COUNT (comments.article_id) AS INT) AS comment_count
+  FROM articles
+  LEFT JOIN comments ON comments.article_id = articles.article_id
+  GROUP BY articles.article_id
+  ORDER BY articles.created_at DESC`;
```
Comment (nc-nick-jones) [inline models/articles.model.js:38]: really lovely SQL query.

**Review by nc-nick-jones — COMMENTED** [review]
Really excellent work!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -80,3 +80,34 @@ describe("/api/articles/:article=id", () => {
...
+    return request(app)
+      .get("/api/bananana")
+      .expect(404)
+      .then((response) => {
+        expect(response.status).toBe(404);
+      });
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:111]: we need a test to check articles are in descending date order

**Review by nc-nick-jones — COMMENTED** [review]
nice work! Just a bit to add to the tests for this one.  Otherwise looking great!

File: __tests__/app.test.js (hunk 3 above)
Comment (ruchikaroy) [reply __tests__/app.test.js:111]: added test for the same, thank you 
https://github.com/ruchikaroy/nc-news/pull/new/task5-PR-review-comments

**Pull request merged**